db/migrate/20220302163416_change_location_id_type_from_announcements.rb

```diff
@@ -0,0 +1,9 @@
+class ChangeLocationIdTypeFromAnnouncements < ActiveRecord::Migration[6.0]
+  def up
+    change_column :announcements, :location_id, :integer
```

Comment:
Yes! Dropping and adding in Rails Migration would be methods "up" and "down" -- they're used to specify what needs to be executed in the forward change and reverse for this migration. There is also a "change" method which basically does both within it, but I decided to use "up" and "down" because it is safer.

Reply:
We don't have any announcements so we can let this fly.